app/main/views/services.py

```diff
@@ -233,11 +241,16 @@ def import_service(service_id):
         abort(400, "Database Error: {0}".format(e))
 
     if not framework.expired:
+        try:
             search_api_client.index(
                 service_id,
                 service.data,
                 supplier.name,
                 framework.name)
+        except apiclient.HTTPError as e:
+            current_app.logger.warning(
```

Comment:
Don't know if we should log these as error rather than warning? Something is definitely wrong if the index updates are failing.